aicsimageio/readers/czi_reader.py

```diff
@@ -183,6 +188,27 @@ def scenes(self) -> Tuple[str, ...]:
                 # If the scene is implicit just assign it name Scene:0
                 if len(scene_names) < 1:
                     scene_names = [metadata_utils.generate_ome_image_id(0)]
+                else:
+                    # reconcile scene list against the dims shape
+                    dims_shape = czi.get_dims_shape()
+                    if len(scene_names) != len(dims_shape) and czi.shape_is_consistent:
+                        dims_shape_dict = dims_shape[0]
+                        scene_range = dims_shape_dict.get(CZI_SCENE_DIM_CHAR)
+                        if scene_range is not None:
+                            scene_names = scene_names[scene_range[0] : scene_range[1]]
+                        else:
+                            # If this is the root node of a split multiscene czi,
+                            # then the scene_range could be None because the dims_shape
+                            # will be effectively empty.
+                            # We do not currently support loading multi-file split
+                            # scene CZI files
+                            log.warning(
+                                "CZI file appears to contain multiple scenes but "
+                                "dimension data is not available in this file. "
+                                "Root node of split multi-scene CZI files are not "
+                                "supported by CziReader."
+                            )
```

Comment:
Shouldn't this raise? If this file isn't supported it should fully fail no?

Reply:
A previous commit had this as a raise, but:  with a multiscene split-file czi, the parent file can accurately report the list of scenes even though it doesn't have any pixel data.  So I warn if scenes are requested, and that should let you know that if you try to get_image_data it's not going to work.  This way you can at least open the parent file and get some info out of it without crashing.

Reply:
Idk if that's true though. If there is no pixel data then even a call to `reader.metadata` will fail I think...

Reply:
creating the aicsimage object and calling .scenes worked with this code